backend/frontend/src/App.js

```diff
@@ -1,67 +1,63 @@
-import React from 'react'
-import { Container } from 'react-bootstrap'
-import { HashRouter as Router, Route, Routes } from 'react-router-dom'
-import Header from './components/Header'
-import Footer from './components/Footer'
+import React from "react";
```

Comment:
nie wiem czy trzymanie całego frontu w katalogu backend to dobry pomysł

Reply:
czyli lepiej przenieść go do root folderu?

Reply:
wg mnie tak

Reply:
done